test/testdata/bootstrappolicy/bootstrap_cluster_roles.yaml

```diff
@@ -62,6 +62,7 @@ items:
     - persistentvolumes/status
```

Comment:
There two more commits 'DaemonSet controller needs to be able to get daemonsets' and 'Allow deployer to watch replication controllers' touching those. How about squashing all of that into one commit?